spec/acceptance/class_spec.rb

```diff
@@ -15,11 +15,8 @@
   context 'default class inclusion' do
     let(:pp) do
       <<-EOS
+      class { 'erlang': repo_source => 'packagecloud' } ->
```

Comment:
https://github.com/voxpupuli/puppet-erlang/blob/master/REFERENCE.md#repo_source This is not valid on all OSes (and I think may be default anyway). I'm testing taking it out and leaving the default here.